.env.dist

```diff
@@ -0,0 +1,25 @@
+###> symfony/framework-bundle ###
+APP_ENV=dev
+URL=local.appbuild.com
+JAWSDB_URL=mysql://appbuild_usr:appbuild_pwd@local.appbuild-db.com/appbuild_db
+SYMFONY_SECRET=ThisTokenIsNotSoSecretChangeIt
+DEFAULT_LOCALE=en
+TIMEZONE=UTC
+SESSION_SAVE_PATH=%kernel.project_dir%/var/sessions/%kernel.environment%
+BUILDS_APPLICATION_DIR=%kernel.project_dir%/var/build_files
+
+###> symfony/swiftmailer-bundle ###
+MAILER_TRANSPORT=smtp
+MAILER_HOST=127.0.0.1
+MAILER_USER=null
+MAILER_PASSWORD=null
+
+###> Webpack ###
```

Comment:
```suggestion
# webpack-dev-server (to watch assets modification in development environment)
```

`###>` / `###<` syntax only works with `symfony/flex`, so Webpack is not concerned